### PR TITLE
Made card one component

### PR DESCRIPTION
**Pull request by sukhrajghuman**
Need help getting card height same plz

### What this PR does

cleaned up css

### Checklist
- [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: src/components/Campaign.js [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
+import Card from "./Card";
 
 const Campaign = () => (
 	<div className="container">
 		<div className="row">
-			<div className="col-xl-2 col-md-3">
-				<CampaignTile
-					href="https://jobactive.gov.au/path"
-					imgSrc="https://www.australia.gov.au/sites/default/files/styles/homepage_quick_link_carousel/public/global_site/library/images/carousel_images/ausgovau_YouthJobsPaTH_259x153.jpg?itok=goqS2o8J"
+			<div className="col-md-3">
```
Comment (azerella) [inline src/components/Campaign.js]: I think we should have the `col-xl-2` tag in all divs here

File: src/components/Card.scss [hunk 2]
```diff
@@ -1,47 +1,50 @@
...
 
-		&.card--purple-light {
-			border-top: solid 5px $purple-light;
-			&:hover {
-				background: $purple-light-hover;
-			}
-		}
+a.card {
+	box-shadow: 0 0.25rem 0.5rem 0 rgba(0, 0, 0, 0.2);
+	transition: 0.3s;
```
Comment (azerella) [inline src/components/Card.scss]: As per review from @alex-page we're removing all animation elements e.g `transition` for a11y

File: src/components/Card.scss [hunk 3]
```diff
@@ -1,47 +1,50 @@
...
+	}
 
-		&.card--purple {
-			border-top: solid 5px $purple;
-			&:hover {
-				background: $purple-hover;
-			}
-		}
+	&:hover {
+		transition: 0.5s;
```
Comment (azerella) [inline src/components/Card.scss]: Same here

**Review by azerella — CHANGES_REQUESTED** [review]
I think for now just adding a `min-height` property your happy with is a suitable MVP.

File: src/components/Campaign.js (hunk 1 above)
Comment (sukhrajghuman) [reply src/components/Campaign.js]: So it looks like this?

image

Comment (azerella) [reply src/components/Campaign.js]: It should go from: 
image

to quite quickly:

image

File: src/pages/index.js [hunk 4]
```diff
@@ -10,14 +10,14 @@ import "./index.scss";
 class Index extends React.Component {
 	render() {
 		return (
-			<div className="index">
-				<div className="container-fluid">
+			<div>
+				<div className=" container au-body">
```
Comment (azerella) [inline src/pages/index.js]: Empty `className`?

**Review by azerella — APPROVED** [review]

**Pull request merged**